modules/oe_theme_helper/tests/src/Unit/TwigExtensionTest.php

```diff
@@ -7,17 +7,17 @@
 use Drupal\Core\Language\LanguageInterface;
 use Drupal\Core\Language\LanguageManagerInterface;
 use Drupal\Core\Template\Loader\StringLoader;
-use Drupal\oe_theme_helper\TwigExtension\Filters;
+use Drupal\oe_theme_helper\TwigExtension\TwigExtension;
 use Drupal\Tests\UnitTestCase;
 
 /**
  * Tests for the custom Twig filters extension.
  *
  * @group oe_theme_helper
  *
- * @coversDefaultClass \Drupal\oe_theme_helper\TwigExtension\Filters
+ * @coversDefaultClass \Drupal\oe_theme_helper\TwigExtension\TwigExtension
  */
-class TwigFiltersTest extends UnitTestCase {
+class TwigExtensionTest extends UnitTestCase {
```

Comment:
As I can see here missed test coverage for new toEclIcon() method.
@ademarco do we need to do this within this PR?

Reply:
Yeah maybe let's just add a very simple one that tests that non existing icon will get the default result.